docs/topic-guides/system-concepts/resource-pool.txt

```diff
@@ -97,22 +114,30 @@ resource pool the agent should join.
 *****************************
 
 With the introduction of resource pools, the :ref:`master-configuration`
-format has changed to a new format. The old format had the top level
-fields of ``scheduler`` and ``provisioner`` which set the scheduler and
-provisioner settings for the cluster.
-
-The new format has the top level fields ``resource_manager`` and
-``resource_pools``. The ``resource_manager`` section is for cluster
+format has changed to a new format.
+
+This is a backwards compatible change and cluster configurations in the
+old format will continue to work. A configuration in the old format is
+interpreted as a cluster with a single resource pool that is the default
+for both CPU and GPU tasks. However, to take full advantage of resource
+pools, you will need to convert to the new format, which is a simple
+process of moving around and renaming a small number of top-level
+fields.
+
+The old format had the top level fields of ``scheduler`` and
```

Comment:
nonblocking: probably should make it a separate section ``migration guide`` in the cluster configuration reference not in a topic guide and link it in the release note.

Reply:
Probably a good idea. If you have time to get it into this release, that would be great. Otherwise we can do that for the next release.